vowpalwabbit/parse_args.cc

```diff
@@ -1688,6 +1688,38 @@ void free_args(int argc, char* argv[])
   free(argv);
 }
 
+void print_progressive_validation_header(vw& all)
```

Comment:
Can this be const?

Reply:
don't think so since we're using trace_message